src/Tizen.Applications.EventManager/Tizen.Applications/SystemEvents.cs

```diff
@@ -0,0 +1,898 @@
+﻿/*
+ * Copyright (c) 2019 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+namespace Tizen.Applications.EventManager.SystemEvents
```

Comment:
How to use it?

Reply:
Not required. I will remote it

Reply:
Could you suggest me the proper structure?

Reply:
You did't provide any information to use SystemEvents. 
You only provide a key strings, I have no idea how to use it

Reply:
``` c#
public static class LaunchEvent {
     public static LaunchedEventArgs GetLaunchedEventFromAppControl(AppControl appcontrol);
}

public class LaunchedEventArgs {
    public string EventSourceUri {get;}
    public string Data {get;}
}

```

Reply:
I just want to provide strings of system event as the link below
https://developer.tizen.org/development/api-references/native-application?redirect=https://developer.tizen.org/dev-guide/5.0.0/org.tizen.native.mobile.apireference/group__CAPI__EVENT__MODULE.html

Reply:
How to subscribe System event?

Reply:
It is the same as application event. 
Then, Should I change 'ApplicationEventReceiver' class to 'EventReceiver'?